tests/client/dataset_client_test.py

```diff
@@ -65,6 +65,13 @@ def test_create_dataset_model(scicat_client, derived_dataset):
             assert expected == dict(downloaded)[key], f"key = {key}"
 
 
+def test_validate_dataset_model(real_client, require_scicat_backend, derived_dataset):
+    real_client.scicat.validate_dataset_model(derived_dataset)
+    derived_dataset.type = "banana"
```

Comment:
Please use a different field. E.g., set the `contactEmail` to something that is not an email address.